tools/dockerfile-llb-frontend/test/docker/docker_test.go

```diff
@@ -0,0 +1,83 @@
+package docker
+
```

Comment:
Can you please add a copyright preamble to the top of this file too?

Reply:
Done